packages/relay-compiler/RelayCompilerPublic.js

```diff
@@ -30,6 +31,8 @@ const {
 
 export type {CompileResult, ParserConfig, WriterConfig} from 'graphql-compiler';
 
+const RelayJSModuleParser = RelaySourceModuleParser(FindGraphQLTags.find);
+
 module.exports = {
```

Comment:
so this interface remains the same, without being parametrized by a language, correct? this means that the only way to use plugins is via the cli, correct?

Reply:
That is currently correct.